### DISCUSSION TITLE
GUI Like AutoHotkey

### DISCUSSION THREAD

**Discussion by MusoCity** [Hammerspoon/hammerspoon#3629] [Q&A]
With AutoHotkey I can create a GUI and add some function buttons to call a function to access an app's menu etc.., it can be set to AlwaysOnTop:
Gui, +AlwaysOnTop
Gui, Add, Button, x10 y50 w70 h25 gFunction1, Paste
Gui, Show, w200 h100

Function1:
WinMenuSelectItem, ahk_class Notepad, , Edit, Paste

Return

How can I do this with Lua in HammerSpoon ???

**Comment by MusoCity** [comment]
Maybe there is another forum proper for Hammerspoon ? This one here https://www.autohotkey.com/boards/index.php you get help right away. It seems pretty dead around here, maybe I'm just talking to myself :(

**Reply by latenitefilms** [reply]
I just use webview's for [CommandPost](https://commandpost.io)'s user interface - all the UI is HTML, which triggers Lua.

You could also look at:

https://github.com/asmagill/hs._asm.uitk
